packages/decimal/src/lib.rs

```diff
@@ -861,4 +862,58 @@ mod tests {
     fn decimal_pow_overflow_panics() {
         _ = Decimal::MAX.pow(2u32);
     }
+
+    #[test]
+    fn decimal_abs_with_negative_number() {
+        let decimal = Decimal::new(-128);
+
+        assert_eq!(decimal.abs(), Decimal { 0: 128 });
+    }
```

Comment:
Decimal should not have a negative number in this implementation.

Reply:
meaning the test is completely useless?